cmd/root.go

```diff
@@ -94,13 +105,18 @@ func processBrokerOpts(cmd *cobra.Command, args []string) error {
 func init() {
 	rootCmd.Flags().SortFlags = false
 
+	// main command flags
 	rootCmd.Flags().StringVar(&namespace, "namespace", "default", "Watch for objects in desired namespace")
 	rootCmd.Flags().StringVar(&metricsAddr, "metrics-addr", ":8080", "Metrics endpoint will bind to this address")
+	// NOTE: we can probably pass this image/tag in as a build flag
```

Comment:
Maybe as a default, but we will need to configure it for offline scenarios

Reply:
Yup. in-sync